src/components/templates/PosterHall/PosterHall.tsx

```diff
@@ -43,6 +43,7 @@ export const PosterHall: React.FC<PosterHallProps> = ({ venue }) => {
   const { liveVenueEvents } = usePosterHallSubVenues(venue.id);
 
   const renderedSubvenues = useMemo(() => {
+    if (!liveVenueEvents) return;
     return liveVenueEvents.map((subVenueEvent) => (
```

Comment:
Remember to always include a blank line of space after a 'guard `if`' like this.

Reply:
Thanks! Will do in the future — this `if` statement has been removed in the current version

Reply:
Thanks! implemented in 3ac268ed04961eb5ee453b9a5cc33cf3893a0da3